core/src/organization/member-restriction/member-restriction.service.ts

```diff
@@ -73,7 +87,24 @@ export class MemberRestrictionService {
         
         memberRestriction = this.memberRestrictionRepository.merge(memberRestriction, {manualExpiration, manualExpirationReason});
         await this.memberRestrictionRepository.save(memberRestriction);
-        
+
+        // This is the message we'll send to the front end about the manual
+        // expiration
+        const newMember = await this.memberService.getMemberById(memberRestriction.memberId);
+        const eventPayload = {
+            id: memberRestriction.id,
+            eventType: 2,
```

Comment:
What is event type?

Reply:
Could probably make it an enum, literally 'restriction created' (1) or 'restriction expired' (2)